docs/readme.md

```diff
@@ -1 +1,22 @@
-- placeholder -
+## Plugin Install Instructions
+
+
+### Supported IDEs
+
+
+The Flutter Plugin can be installed on following IntelliJ-based products:
+
+* IntelliJ 2016.2+ ([Ultimate or Community](https://www.jetbrains.com/idea/download/))
+
+
+### Installation
```

Comment:
Ah right.  I was scoping this just to installation but it presupposes Dart is installed (and configured).  I'll inline those instructions.